.pylintrc

```diff
@@ -0,0 +1,13 @@
+[FORMAT]
+indent-string='    '
+max-line-length=100
+
+[BASIC]
+function-naming-style=camelCase
+argument-naming-style=camelCase
+variable-naming-style=camelCase
+method-naming-style=camelCase
+attr-naming-style=camelCase
+
```

Comment:
Maybe we could relax the design requirements, e.g. fixing a larger maximum number of function arguments or local variables and so on
```
[DESIGN]
max-args=10
max-locals=40
....
```
In my experience methods that exceeds the default configuration could be fine/necessary. @hipe4ml/developers What do you think?

Reply:
> @mconcas To me the very granular set of tests is fine. However, I do not get the difference between pylint and flake8 and if both tests are steered by `.pylintrc`

They have a great region of overlap, with the first run they both raised 5 errors each different each other. Eventually we could also consider to remove one of them, for the moment I would try to give them a chance.
AFAIK only pylint relies on `.pylintrc`

Reply:
Might definitely be the case. Not a strong opinion.